container/examples/container-iframe/index.html

```diff
@@ -5,12 +5,27 @@
     <title>Luigi iFrame Container</title>
   </head>
 
+  <body>
```

Comment:
Seems like somehow this file was not prettified. could you double check to run prettier on it as well. Make sure to have run npm install on root folder to apply our recent prettier config changes